server.js

```diff
@@ -2,49 +2,74 @@
 
 var request = require('request')
 var express = require('express')
+var xml2js = require('xml2js')
+var coberturaPath = '/jenkins/cobertura/'
+var jacocoPath = '/jenkins/jacoco/'
 
 var app = express()
+var badgeColor = function(cov) {
+	if (cov < 20) {
+		return 'red'
+	} else if (cov < 80) {
+		return 'yellow'
+	} else {
+		return 'brightgreen'
+	}
+}
+var getBadgeUrl = function(req, label, color) {
+	var badgeUrl = 'https://img.shields.io/badge/coverage-' + label + '-' + color + '.svg'
+	var style = req.param("style")
```

Comment:
Since the caller of `getBadgeUrl` is already familiar with the nature, and structure, of `req`, it doesn't sound unprecedented to have the caller pull out the `style` property..